src/regex/deductions/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+from django.db.models import Sum
+
+from privates.admin import PrivateMediaMixin
+from regex.utils.views.private_media import PrivateMediaView
+
+from .models import Deduction
+
+
+class DeductionPrivateMediaView(PrivateMediaView):
+    model = Deduction
+    permission_required = 'invoices.can_view_invoice'
```

Comment:
Hmm, now that I think about it having the same permission as for invoices doesn't make sense here :thinking: